clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -164,6 +164,11 @@
      */
     private static final long INVALID_SHUTDOWN_TIME = -1;
 
+    /**
+     * Thread name prefix for admin client network thread
+     */
+    public static final String NETWORK_THREAD_PREFIX = "kafka-admin-client-thread";
```

Comment:
Maybe we should have this as package private and expose it via a test class in the same package. We don't want users to depend on it and this class is public API.